projects/core/src/group-skipper/config/group-skipper.config.ts

```diff
@@ -0,0 +1,11 @@
+export abstract class GroupSkipperElement {
+  type: string;
+  id: string;
```

Comment:
`id` seems to be too generic name imho. maybe some more descriptive (name, selector, ..., ?)